powerauth-push-server/src/main/java/io/getlime/push/service/PushSendingWorker.java

```diff
@@ -149,13 +151,10 @@ void sendMessageToAndroid(final FcmClient fcmClient, final PushMessageBody pushM
             callback.didFinishSendingMessage(PushSendingCallback.Result.FAILED);
         };
 
-        // Callback when FCM request succeeds
```

Comment:
🤔 Hmm... why was this here...

Reply:
Yes, however, note that the original Rest client code with `ClientResponse` was a bit more generic - there was full access to the response including HTTP headers, etc. Subscribing on `exchangeToMono` provides `ResponseEntity<T>` or exception where we can gather all the details. This handling should be hopefully fine for our use cases.